documentation/dotnet-trace-instructions.md

```diff
@@ -86,6 +86,12 @@ Microsoft-Windows-DotNETRuntime         | [The Runtime Provider](https://docs.mi
 Microsoft-Windows-DotNETRuntimeRundown  | [The Rundown Provider](https://docs.microsoft.com/en-us/dotnet/framework/performance/clr-etw-providers#the-rundown-provider)<br>[CLR Rundown Keywords](https://docs.microsoft.com/en-us/dotnet/framework/performance/clr-etw-keywords-and-levels#rundown)
 Microsoft-DotNETCore-SampleProfiler     | Enable the sample profiler
 
+## `dotnet-trace` output formats
+
+You can change the output file format using the `-f|--format` option. ou can currently choose between `netperf` (the default on Windows) and `speedscope` (the default on all other OSes). Speedscope files can be opened at https://www.speedscope.app.
```

Comment:
Should it be: [speedscope](https://www.speedscope.app)

Reply:
In this case I think it makes sense to show that it is a web app with the visible URL.